_posts/2024-10-17-uswds-3.9.0.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Out now: USWDS 3.9.0"
+tags:
+- USWDS release
+category: About
+excerpt_link_url: https://github.com/uswds/uswds/releases/tag/v3.9.0
```

Comment:
Would `post_url`  be more accurate since there's no excerpt? 

```suggestion
post_url: https://github.com/uswds/uswds/releases/tag/v3.9.0
```

Reply:
I changed it to `custom_url` in https://github.com/uswds/uswds-site/pull/2979/commits/c4d016a972835cafeb014ae58c443dc913828ca4 since that felt a bit clearer to me. Let me know if that isn't intuitive to you.

Reply:
@amyleadem the name doesn't quite match its function. Maybe `preview_url` or `outgoing_url`?

Reply:
Updated to `preview_url` in [0a1b612](https://github.com/uswds/uswds-site/pull/2979/commits/0a1b6127744ad55d53158708c5a1becba1760843)